### DISCUSSION TITLE
Is there any way to specify StartRow as we can using Search Rest API?

### DISCUSSION THREAD

**Discussion by dmoorjmalani** [microsoft-search/pnp-modern-search#4032] [Q&A]
Hi - I am looking to skip certain number of results, and i believe that can be done using startrow parameter using Search API. Is there any setting on PnP Search Results to do the same?

https://learn.microsoft.com/en-us/sharepoint/dev/general-development/sharepoint-search-rest-api-overview#startrow

**Comment by wobba** [comment]
Not currently implemented. Can you explain your scenario?

**Comment by dmoorjmalani** [comment]
Hi @wobba - We have two search results webparts on the screen with a same query, one we have configured to use a slider webpart and just shows 5 results and other is using cards layout and we want it not to show those first 5 results covered by the slider.
